dist/images/Dockerfile

```diff
@@ -0,0 +1,51 @@
+#
+# This is the OpenShift sdn-ovn image.
```

Comment:
"OpenShift sdn-ovn image" -> "ovn-kubernetes image".

Reply:
Clayton suggested origin-network-ovn

Reply:
@pecameron yeah, this is upstream ovn-kube though, so Clayton loses...  The Origin-specific YAML files might use that though.